library/junos_install_os

```diff
@@ -155,7 +161,20 @@ import logging
 import re
 import os
 from distutils.version import LooseVersion
+from ansible.module_utils.six import string_types
+from ansible.module_utils.basic import *
 
+def _convert_to_boolean(val):
+    if val is None or isinstance(val, bool):
+        return val
+    if isinstance(val, string_types):
+        val = val.lower()
+    if val in BOOLEANS_TRUE:
+        return True
+    elif val in BOOLEANS_FALSE:
+        return False
+    else:
+        raise TypeError
```

Comment:
I dont its good idea to raise error for other data type and then catch error to set non boolean value in line 247.